frontend/src/features/properties/map/propertyInformation/PropertyContainer.tsx

```diff
@@ -0,0 +1,96 @@
+import { usePropertyDetails } from 'features/mapSideBar/hooks/usePropertyDetails';
+import {
+  InventoryTabNames,
+  InventoryTabs,
+  TabInventoryView,
+} from 'features/mapSideBar/tabs/InventoryTabs';
+import LtsaTabView from 'features/mapSideBar/tabs/ltsa/LtsaTabView';
+import PropertyAssociationTabView from 'features/mapSideBar/tabs/propertyAssociations/PropertyAssociationTabView';
+import { PropertyDetailsTabView } from 'features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView';
+import React, { useState } from 'react';
+
+import ComposedProperty from './ComposedProperty';
+
+export interface IPropertyContainerProps {
+  composedProperty: ComposedProperty;
+  setEditMode: (isEditing: boolean) => void;
+}
+
+/**
+ * container responsible for logic related to map sidebar display. Synchronizes the state of the parcel detail forms with the corresponding query parameters (push/pull).
+ */
+export const PropertyContainer: React.FunctionComponent<IPropertyContainerProps> = ({
+  composedProperty,
+  setEditMode,
+}) => {
+  const showPropertyInfoTab = composedProperty.apiProperty !== undefined;
+
+  const tabViews: TabInventoryView[] = [];
+
+  tabViews.push({
+    content: (
+      <LtsaTabView
+        ltsaData={composedProperty.ltsaData}
+        ltsaRequestedOn={composedProperty.ltsaDataRequestedOn}
+        loading={composedProperty.ltsaLoading}
+      />
+    ),
+    key: InventoryTabNames.title,
+    name: 'Title',
+  });
+
+  tabViews.push({
+    content: <></>,
+    key: InventoryTabNames.value,
+    name: 'Value',
+  });
+
+  var defaultTab = InventoryTabNames.title;
+
+  // TODO: this should have a loading flag
+  const propertyViewForm = usePropertyDetails(composedProperty.apiProperty);
```

Comment:
Does this todo make sense? it looks like the PropertyDetailsTabView is being sent a loading flag that corresponds to this object. Maybe I don't understand.

Reply:
Yes. Unfortunately, this causes the page to stutter when loading information. The loading screen goes away once apiProperty is retrieved, but updated again once usePropertyDetails completes since that one retrieves extra information. IMO this should follow the same pattern of retrieving the information alongside a isLoading